src/components/Dashboard/MainInput/MainInput.jsx

```diff
@@ -54,27 +36,22 @@ export default class MainInput extends React.Component {
     });
   };
 
-  setInputValue = (value) => {
+  setInputValue = (value = '') => {
     const { newPriority } = this.state;
-    const inputDataArr = value.split('');
+    const inputDataArr = value;
     const hashCheck = inputDataArr[inputDataArr.length - 2];
     let setPriority = newPriority;
     if (hashCheck === '#') {
       const getPriority = inputDataArr[inputDataArr.length - 1];
-      if (getPriority === '1') {
-        setPriority = 'High';
-      } else if (getPriority === '3') {
-        setPriority = 'Low';
-      } else {
-        setPriority = 'Medium';
-      }
+      setPriority = changePriority(getPriority);
```

Comment:
not really understandable variable/func names, I would suggest something like
```
getPriority => enteredPriority/priorityIndex
changePriority => getPriority
setPriority  => newPriority
```

Reply:
It was changed.